tests/test_dispersion.py

```diff
@@ -0,0 +1,125 @@
+import io
+import sys
+
+import generate_tempfiles
+import numpy as np
+from calcos import dispersion
+
+
+def test_init():
```

Comment:
Should this be named test since it isn't testing anything?

Reply:
Yes, because we want pytest to run it so that it initializes all the fields needed for testing the actual test functions.

Reply:
I don't believe it's making the object global though, correct? It would need to be called for each instance that disp_obj is needed